packages/server/src/utils/LocalFileSystemHandler.js

```diff
@@ -0,0 +1,124 @@
+
+const fs = require('fs')
+const path = require('path')
+const rimraf = require('rimraf');
+
+/**
+ * Store files in local file system (app-store/packages/server/upload)
+ */
+module.exports = class LocalFileSystemHandler {
+
+    constructor() {
+        this._uploadDirectory = __dirname + '/../../upload' //app-store/packages/server/upload
+        this.createDirectoryIfNotExists(this._uploadDirectory)
+    }
+
+    get directory() {
+        return this._uploadDirectory
+    }
+
+    createDirectoryIfNotExists(dir) {
+        if (!fs.existsSync(dir)) {
+            fs.mkdirSync(dir, { recursive: true, mode: '0744' });
+        }
+    }
+
+
+    saveFile(directoryPath, filename, buffer) {
+        return new Promise((resolve, reject) => {
+            try {
+                this.verifyPath(directoryPath)
+                this.createDirectoryIfNotExists(path.join(this._uploadDirectory, directoryPath))
+            } catch ( err ) {
+                reject(err)
+            }
+            
+            fs.writeFile(path.join(this._uploadDirectory, directoryPath, filename), buffer, (err) => {
+                if ( err ) {
+                    reject(err)
+                } else {
+                    resolve()
+                }
+            })
+        })
+    }
+
+    /**
+     * Downloads the binary data for a file
+     * @param {*} directoryPath
+     * @param {*} filename
+     */
+    getFile(directoryPath, filename) {
+        return new Promise((resolve, reject) => {
+            try {
+                this.verifyPath(directoryPath)
+            } catch ( err ) {
+                reject(err)
+            }
+
+            fs.readFile(path.join(this._uploadDirectory, directoryPath, filename), (err, data) => {
+                if ( err ) {
+                    console.log(err)
+                    reject(err)
+                } else {
+                    resolve({ Body:data })
```

Comment:
You can just change the parameter name to `Body` and then do `resolve({ Body })`, besides that, there's a spacing missing, making this assignment look a bit weird

Reply:
would be possible, but this is essentially just to return on the same format as the AWS api with the data in a Body property, I'd like to keep the name 'data' for the callback parameter as I believe that's cleaner and that you also see the assignment in the resolve call. Agree about the whitespace though